### PR TITLE
Added FSLeyes popup that will display the segmentation ROI  

### LINKED ISSUE #49: FSLeyes popup during realtime_zhim to show the SCT segmentation ROI
The SCT is called to generate an ROI in a phantom or segment the spinal cord in-vivo. Some visual QA is needed. We could have FSLeyes open up to display the ROI volume (which is already generated in nifti format by SCT) overlaid with the GRE volume (however, these images are currently in MaRdI or dicom format, so they would also have to be saved as niftis). One of the overlaid views (ex: sagittal) could additionally be saved as a jpeg.

### PR DESCRIPTION
**Pull request by evaalonsoortiz**
Fixes #49

### CODE REVIEW

File: misc/MaRdI.m [hunk 1]
```diff
@@ -2607,7 +2607,7 @@
 DEFAULT_ISFORCINGOVERWRITE = false ;
 DEFAULT_ISUSINGPROPSEGCSF  = true ; %use the propseg -CSF option
 DEFAULT_CENTERLINEMETHOD   = 'midfov' ;
-DEFAULT_CYLINDERSIZE       = 40 ;
+DEFAULT_CYLINDERSIZE       = 80 ;
```
Comment (jcohenadad) [inline misc/MaRdI.m:2610]: i would specify the unit (vox or mm)

File: misc/MaRdI.m [hunk 2]
```diff
@@ -2684,15 +2684,20 @@
...
-% move segmentation + weights
+% move segmentation, weights + t2* images
 system( ['mv ' Params.tmpSaveDir 't2s_seg.nii ' Params.dataSaveDir 'gre_seg.nii'] ) ;  
-system( ['mv ' Params.tmpSaveDir 't2s_weights.nii ' Params.dataSaveDir 'gre_weights.nii'] ) ;  
+system( ['mv ' Params.tmpSaveDir 't2s_weights.nii ' Params.dataSaveDir 'gre_weights.nii'] ) ; 
+system( ['mv ' Params.tmpSaveDir 't2s.nii ' Params.dataSaveDir 'gre.nii'] ) ; 
 system( ['rm -r ' Params.tmpSaveDir] ) ;
 
+% load FSLeyes to see segmentation overlaid on gre images
+system(['fsleyes ' Params.dataSaveDir 'gre.nii ' Params.dataSaveDir 'gre_seg.nii &' ]);
```
Comment (jcohenadad) [inline misc/MaRdI.m]: to automatically get a red mask (minimize confusion if your grayscale image is high and close to white) with 70% opacity you could do:
~~~
system(['fsleyes ' Params.dataSaveDir 'gre.nii ' Params.dataSaveDir 'gre_seg.nii -cm red -a 70.0 &' ]);
~~~

File: misc/MaRdI.m [hunk 3]
```diff
@@ -2684,15 +2684,20 @@
...
+
 Mask = load_untouch_nii( [ Params.dataSaveDir 'gre_seg.nii' ] );
 mask = Mask.img ;
 mask = logical(permute( mask, [2 1 3] )) ;
```
Comment (jcohenadad) [inline misc/MaRdI.m:2708]: i would add a comment explaining why we need this permutation (unclear if someone else is reading the code)

File: misc/MaRdI.m (hunk 1 above)
Comment (jcohenadad) [reply misc/MaRdI.m:2610]: ok it's actually mm

Comment (evaalonsoortiz) [reply misc/MaRdI.m:2610]: I've added a command prompt that will ask the user to specify the size (in mm)

File: misc/MaRdI.m (hunk 2 above)
Comment (evaalonsoortiz) [reply misc/MaRdI.m]: done

File: misc/MaRdI.m (hunk 3 above)
Comment (evaalonsoortiz) [reply misc/MaRdI.m:2708]: done

Comment (jcohenadad) [reply misc/MaRdI.m:2708]: i don't see the comment

Comment (po09i) [reply misc/MaRdI.m:2708]: me neither

File: misc/MaRdI.m (hunk 1 above)
Comment (jcohenadad) [reply misc/MaRdI.m]: @rtopfer pls comment

File: misc/MaRdI.m (hunk 3 above)
Comment (jcohenadad) [reply misc/MaRdI.m:2708]: @rtopfer pls comment

File: misc/MaRdI.m [hunk 4]
```diff
@@ -2684,15 +2684,20 @@
...
+% move segmentation, weights + t2* images
 system( ['mv ' Params.tmpSaveDir 't2s_seg.nii ' Params.dataSaveDir 'gre_seg.nii'] ) ;  
-system( ['mv ' Params.tmpSaveDir 't2s_weights.nii ' Params.dataSaveDir 'gre_weights.nii'] ) ;  
+system( ['mv ' Params.tmpSaveDir 't2s_weights.nii ' Params.dataSaveDir 'gre_weights.nii'] ) ; 
+system( ['mv ' Params.tmpSaveDir 't2s.nii ' Params.dataSaveDir 'gre.nii'] ) ; 
 system( ['rm -r ' Params.tmpSaveDir] ) ;
 
+% load FSLeyes to see segmentation overlaid on gre images
+system(['fsleyes ' Params.dataSaveDir 'gre.nii ' Params.dataSaveDir 'gre_seg.nii &' ]);
+
```
Comment (rtopfer) [inline misc/MaRdI.m]: % permutation seems to be required after nii load to restore original orientation (prior to nii save)

**Review by jcohenadad — APPROVED** [review]

**Review by po09i — APPROVED** [review]

**Pull request merged**